spring-cloud-gcp-pubsub/src/main/java/org/springframework/cloud/gcp/pubsub/core/PubSubTemplate.java

```diff
@@ -132,10 +134,16 @@ public PubSubTemplate setMessageConverter(PubSubMessageConverter messageConverte
 	}
 
 	@Override
+	@Deprecated
```

Comment:
I believe you still need a `@SupressWarning("deprecation")` since you call a deprecated `pubSubSubscriberTemplate.subscribe()`

Reply:
The  compiler doesn't seem to be complaining.

Reply:
OK. We'll see.
That's why I don't like Maven: too noise during build - we may just miss some simple warning about deprecation violation...